lldb/source/Plugins/Language/Swift/SwiftOptional.cpp

```diff
@@ -44,24 +44,46 @@ static PointerOrSP
 ExtractSomeIfAny(ValueObject *optional,
                  bool synthetic_value = false) {
   if (!optional)
-    return nullptr;
+    return PointerOrSP("Can't get value from null optional");
```

Comment:
I'm not sure how/when this can happen, but if a user sees this message they'll probably be confused at "null optional" which is kind of a nonsensical term in Swift.

Reply:
Yeah, I can't think of good messages for any of these really.  I know how we fail now (because this part of the enum inspection is unimplemented) but once that works it should only be other bugs or broken objects.  I'm not sure we can really say much more than "this didn't work".  The error strings are mostly so I can look at the result and tell where it failed...